src/bundle/Resources/views/themes/admin/ui/component/collapse.html.twig

```diff
@@ -28,7 +28,7 @@
                 {% if extra_actions is defined %}
                     {% for action in extra_actions %}
                         <button type="button" class="ibexa-collapse__extra-action-button btn ibexa-btn ibexa-btn--no-text {{ action.button_class|default('') }}">
-                            <svg class="ibexa-icon ibexa-icon--small">
+                            <svg class="ibexa-icon {{ action.icon_size_class|default('ibexa-icon--small') }}">
```

Comment:
Do you think it would be better to do
```
<svg class="{{ action.icon_class|default('ibexa-icon ibexa-icon--small') }}">
```
and later pass `'icon_class': 'ibexa-icon ibexa-icon--tiny-small',` ? Or treat `ibexa-icon` as class that is always and only change `icon_size_class` to `icon_class`?